### ISSUE TITLE
Unable to customize the Admin Dashboard

### ISSUE THREAD

**Issue by tufail** [strapi/strapi#18864]
Hi , I am trying to change the admin panel using app.js but my changes are not reflecting, I followed the documentation but no luck. 
I tried `yarn build` and 'yarn develop` also but still no luck.
## Bug report
here is my `app.js` code

```
export default {
  config: {
    // Add a new locale, other than 'en'
    locales: [],
    // Override or extend the theme
    theme: {
      // overwrite light theme properties
      light: {
        colors: {
          primary100: "#f6ecfc",
          primary200: "#e0c1f4",
          primary500: "#ac73e6",
          primary600: "#9736e8",
          primary700: "#8312d1",
          danger700: "#b72b1a",
        },
      },
    },
    // Extend the translations
    translations: {
      en: {
        "app.components.LeftMenu.navbrand.title": "Strapi2 Dashboard",
        "app.components.LeftMenu.navbrand.workplace": "Workplace",
      },
    },
    // Disable video tutorials
    tutorials: false,
    // Disable notifications about new Strapi releases
    notifications: { releases: false },
  },

  bootstrap() {},
};

```
 

- Node.js version: 18.18.1
- NPM version: 6
- Strapi version: 4.15.4
- Database: mysql
- Operating system: Windows
- Is your project Javascript or Typescript: Javascript 

### Expected result
Dashboard should show the the title "Strapi2 Dashboard" but still showing "Strapi Dashboard" in left navigation.

**Comment by redondi88** [comment]
I am having the same issue. 
After some comparison I noticed some discrepancies on source files. Let me explain:
I started my project in April this year, Strapi 4.9.0 was the latest then.
I noticed `src/index.js` and `src/admin/*` files are different. Not sure if through 4.9 to 4.15 there have been any breaking changes that  causes this issue.
Hoping someone can identify the actual root cause as I am trying to edit the theme as well.
